.github/workflows/test-and-build.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
           echo "See full release notes at: https://github.com/mongodb-js/vscode/releases/tag/${RELEASE_TAG}\n" > CHANGELOG.md
           npx json -I -f package.json -e "this.version='${RELEASE_VERSION}'"
           npx json -I -f package-lock.json -e "this.version='${RELEASE_VERSION}'"
-        if: startsWith(github.ref, 'refs/tags/')
+        if: ${{ startsWith(github.ref, 'refs/tags/') && runner.os == 'Linux' }}
```

Comment:
Like this you'll get a weird set of assets where for 2 of the platforms you would be able to download a different content from the UI, and i'm not sure that would be obvious.

If the issue was that the build didn't work on windows did you consider trying something like `shell: bash` or even an `if:  !startsWith(github.ref, 'refs/tags/')` on the task that upload assets to avoid any confusion?

Reply:
Ah good call, maybe we could have just one asset built since it should be totally platform independent. I'll add the `shell: bash`, we can always remove the extra platforms if they ever cause confusion.